packages/opencensus-instrumentation-http/src/index.ts

```diff
@@ -15,3 +15,6 @@
  */
 
 export * from './http';
+import {registerAllClientViews, registerAllServerViews, registerAllViews} from './http-stats';
+
+export {registerAllClientViews, registerAllServerViews, registerAllViews};
```

Comment:
Could this just be `export {registerAllClientViews, registerAllServerViews, registerAllViews} from './http-stats';`?

Reply:
Sure, Done.